app/src/main/java/org/mifos/mobilebanking/api/DataManager.java

```diff
@@ -56,161 +56,166 @@ public DataManager(PreferencesHelper preferencesHelper, BaseApiManager baseApiMa
         clientId = this.preferencesHelper.getClientId();
     }
 
-    public Observable<User> login(String username, String password) {
+    public ObservableSource<User> login(String username, String password) {
```

Comment:
@Rish02Sharma Please use  `Observable` and import the RxJava2 class  with name `import io.reactivex.Observable;`